src/containeirs/NoMatch/NoMatch.tsx

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
-import styled from 'styled-components';
+import { Button, ButtonType } from 'components/Button';
+import { Link } from 'react-router-dom';
+import {
+  Wrapper, LeftSide, RightSide, NoMatchAltLoader, NoMatchLoader, NoMatchLogo, NoMatchAppTitle, NoMatchText, LogoNameContainer, RedirectButton,
+} from './styled';
 
-const Wrapper = styled('div')`
-  width: 80%;
-  float:right;
-`;
-
-const NoMatchPage = () => {
+const NoMatch = () => {
   return (
     <Wrapper>
-      <p>No Match</p>
+      <LeftSide>
+        <LogoNameContainer>
+          <NoMatchLogo />
```

Comment:
Czuję że to `NoMatchLogo` może być użyte też w `HomePage` więc dać do components komponent Logo i tam dać całe `styled.ts`, `Logo.tsx` i ułożyć dobrze elementy